src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.felles.FamilieIntegrasjonerClient
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.oppgave.IdentGruppe
+import no.nav.familie.kontrakter.felles.oppgave.OppgaveIdentV2
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.kontrakter.felles.oppgave.OpprettOppgaveRequest
+import no.nav.familie.util.VirkedagerProvider
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+@Service
+class OppgaveService(private val oppgaveClient: OppgaveClient,
```

Comment:
Tenker det kan være fint å gjøre slike endringer separat fra noe som man commiter som "fjerner warnings", med kanskje en beskrivende tekst til hvorfor man gjør den endringen

Reply:
Snakker vi generelt nå, for fremtidige endringer, eller ønsker du at jeg skal revertere denne delen av endringen og gjøre den i en egene PR?

Reply:
Jeg tenker det gjelder alle 3 delene, bedre å gjøre det på en ryddig måte der vi kan skjønne hva det påvirker/om en slik endring egentlige trengs, enn att den fort blir oversett blant alle andre endringer i en PR som denne. 
Også med tanke på min kommentar i https://github.com/navikt/familie-ef-iverksett/pull/182#discussion_r724749633